extensions/kusto/config.json

```diff
@@ -0,0 +1,26 @@
+{
+	"downloadUrl": "https://github.com/Microsoft/sqltoolsservice/releases/download/v{#version#}/microsoft.sqltools.servicelayer-{#fileName#}",
```

Comment:
I wonder if it's better to share the same Tools Service binaries the mssql extension downloads to avoid having duplicate binaries on the machine.  I think this is what azure resource provider and credentials already do